### DISCUSSION TITLE
wizard-vicuna instead of codellama?

### DISCUSSION THREAD

**Discussion by rosmur** [rjmacarthy/twinny#75] [Q&A]
Hello, I'd like to use wizard-vicuna instead of codellama. Is this feasible? (I have limited disk space and cant afford a separate coding optimized model..) 

I tried changing the model and i get the error "Sorry I dont understand. Please try again" and a spinning indicator

<img width="268" alt="image" src="https://github.com/rjmacarthy/twinny/assets/89332196/a68c0c97-e9b8-4efb-8f8b-2679ac34d8c8">

I've changed to

**Comment by rjmacarthy** [comment]
Please include more details about which provider you are using, e.g Ollama and the exact model you are trying to use.  Also please include details for the `chat.hbs` template file.

Thanks.

**Comment by rosmur** [comment]
Its Ollama wizard-vicuna-uncensored: https://ollama.com/library/wizard-vicuna-uncensored.

Sorry I dont know how to get the chat.hbs template

**Comment by rjmacarthy** [comment]
It should work by default now using Ollama as twinny now supports OpenAI specification.
